middleware/headers_test.go

```diff
@@ -162,4 +168,8 @@ func TestOnlyPskHeaders(t *testing.T) {
 	if c.Get(h.ACCOUNT_NUMBER).(string) != "9876" {
 		t.Errorf("%v was set as psk-account instead of %v", c.Get(h.ACCOUNT_NUMBER).(string), "9876")
 	}
+
+	if c.Get(h.PSK_USER).(string) != "555555" {
```

Comment:
the message in t.Errorf() is not updated for PSK_USER

Reply:
thanks I updated both